### ISSUE TITLE
Checkbox can't be unchecked within form with method="GET"

### ISSUE THREAD

**Issue by kmmbvnr** [hotwired/turbo#497]
I have several checkboxes as a filters for a page. One or two can be checked and page would be updated successfully, but then, there is no way to uncheck all checkboxes.

```html
<form method="GET">
    <input type="checkbox" name="status" value="NEW" >
    <input type="checkbox" name="status" value="COMPLETED" >
```

hotwire/turbo adds but does not remove `?status=...` param from the requested page URL. Is there any way to overcome this?

_"@hotwired/turbo": "^7.0.1"_

**Comment by seanpdoyle** [comment]
Thank you for opening this issue.

There were some changes shipped between 7.0.1 and [7.1.0](https://github.com/hotwired/turbo/releases) that altered code responsible for encoding `<form method="get">` parameters.

Are you able to reproduce the issue with the latest release?

**Comment by kmmbvnr** [comment]
Yeah, indeed,  7.1.0 helps! Thank you!
